src/main/g8/build.sbt

```diff
@@ -3,11 +3,20 @@ version := "1.0"
 
 organization := "com.lightbend.akka.samples"
 organizationHomepage := Some(url("https://akka.io"))
-licenses := Seq(("CC0", url("https://creativecommons.org/publicdomain/zero/1.0")))
+licenses := Seq(
+  ("CC0", url("https://creativecommons.org/publicdomain/zero/1.0"))
+)
 
-scalaVersion := "2.13.3"
+scalaVersion := "2.13.5"
```

Comment:
```suggestion
scalaVersion := "2.13.6"
```

Reply:
Yeah, there are other versions that are not on the latest available either. I prioritized sync-ing with the samples on the guide rather than keeping it on the bleeding edge.

Reply:
See https://github.com/akka/akka-platform-guide/blob/main/docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/build.sbt

After all the samples on the guide are run and tested on each commit while this produces a scaffolding with a lot of dependencies the code doesn't even use so we can't really know if we're introducing any versioning issue. I'd trust the code on the guide.